src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -267,9 +267,13 @@ case class Seqn(stmts: Vector[Stmt])(val info: Source.Parser.Info) extends Stmt
 
 case class Label(id: LabelProxy)(val info: Source.Parser.Info) extends Stmt
 
+case class Continue(label: Option[String], escLabel: String, invs: Vector[Assertion])(val info: Source.Parser.Info) extends Stmt
+
```

Comment:
Would there be a way for us to not have to include the invariants here? Would it be impractical to get them from the ast somehow?